README.md

```diff
@@ -276,14 +279,16 @@ and in `routes.swift` we'll add:
 router.get ("todos/create", use: frontendTodoController.renderCreate)
 ```
 
+> Note how we're using the `privateContainer` on the `Request` to limit the scope of our form validation.
```

Comment:
Sounds a bit vague. How about something like "... on the `Request` since that is where the field cache is registered. Which is done to ensure the field cache does not outlive the request."